README.md

```diff
@@ -95,17 +92,18 @@ git submodule update --init
 To create a conda environment with all the dependencies for building CIL run the following `bash` script:
 
 ```bash
-sh scripts/create_local_env_for_cil_development.sh -n NUMPY_VERSION -p PYTHON_VERSION -e ENVIRONMENT_NAME
+bash scripts/create_local_env_for_cil_development.sh
 ```
 
 Or with the CIL build and test dependencies:
+
 ```bash
-sh scripts/create_local_env_for_cil_development_tests.sh -n NUMPY_VERSION -p PYTHON_VERSION -e ENVIRONMENT_NAME
+bash scripts/create_local_env_for_cil_development.sh -t
```

Comment:
What is the reason for removing the version options etc.?

Reply:
seems:

- unnecessary to document here
- painful to keep in sync with the script
- easier to ask users to either run script help (`create_local_env_for_cil_development.sh -h`) or (better) read shell scripts they find on the internet before daring to run them
- not at all explained in the documentation here **why** a user might want to set numpy/python versions... probably pretty big omission which should be fixed in a separate PR?

Reply:
Users don't run that script. It's for developers (i.e. us) to create a working environment to build and test in. All of it could be moved to an environment files as well/instead. As I said below, our contribution process/guidelines need working out. This again is something that came out of the user meeting, we'll be going through the cil readme, documentation, https://tomographicimaging.github.io/CIL/, and contribution instructions/guidelines at the developer meeting on Tuesday so I think it's worth discussing them before we make changes.

Reply:
moved to #1590